### ISSUE TITLE
Add pre-configured bookmarks to selected accounts/profiles

### ISSUE THREAD

**Issue by michaelfecher** [Systanics/CloudGlance#20]
**Is your feature request related to a problem? Please describe.**
Considering a multi account/profile setup with a lot of app target deployments, monitoring and management going on.
All my profiles do need a CFN, IAM, etc. and it's cumbersome to add them manually for each profile.

**Describe the solution you'd like**
I'd like to have one or more customizable templates for auto-creating bookmarks when adding a new or updating an existing profile.

**Comment by rehanvdm** [comment]
Yeah makes sense, I was also thinking of a "Share bookmarks with other profile" type of functionality. Because I have profiles that are read-only and others that are admin for the same account, it is a pain having to manage/keep them in synch on both profiles

**Comment by michaelfecher** [comment]
Any update about this?

**Comment by rehanvdm** [comment]
Hey, sorry I have not been giving this project the attention it deserves. I am busy with https://github.com/rehanvdm/serverless-website-analytics and will be for a while still, then I will get back to this. Can't make any promises but maybe end of the year.
